tests/run_tests.py

```diff
@@ -317,7 +351,13 @@ def run_system(config_file, vic_exe, test_data_dir, out_dir, driver):
     # Get setup
     config = read_configobj(config_file)
 
-    # drop invalid driver tests
+    # If multiple drivers, put VIC objects in a dict
+    if isinstance(driver, list):
+        dict_vic_exe = {}
+        for i, d in enumerate(driver):
+            dict_vic_exe[d] = vic_exe[i]
+
+    # Drop invalid driver tests
     config = drop_tests(config, driver)
```

Comment:
since the configuration file specifies the driver(s) for each test, you could use that to get the correct exe from the dictionary of executables.  

A try/except would be how we skip invalid driver tests.

```Python
for driver, vic_exe in drivers.items():
    try:
        config['driver'][driver]
    except KeyError:
        SkipVICTest('skipping test %s' % testname)
```